packages/admin-ui/src/radio/Radio.style.ts

```diff
@@ -0,0 +1,85 @@
+import { focusVisible, style } from '@vtex/admin-ui-core'
+import { get } from '@vtex/admin-ui-util'
+
+export const checkmark = style({
+  content: '""',
+  position: 'absolute',
+  display: 'block',
+  borderRadius: 'circle',
+  top: '0.3rem',
+  size: '0.5rem',
+  bg: '$form.neutral',
+})
+
+export const baseline = style({
+  ...focusVisible('neutral'),
```

Comment:
Shouldn't be main?

Reply:
Same reason: https://github.com/vtex/admin-ui/pull/137#discussion_r849492849